### ISSUE TITLE
[:bug:] `lambda setup` fails because of duplicate bucket name

### ISSUE THREAD

**Issue by zwily** [Testlio/lambda-tools#60]
**Type:**
<:bug:>

**Description:**
When running `lambda setup`, it fails with this error:

```
Creating S3 bucket: lambda-tools-3-assets-us-east-1 ✖ The requested bucket name is not available. The bucket namespace is shared by all users of the
```

It looks like it's attempting to create a bucket named `lambda-tools-3-assets-us-east-1`. Bucket names must be globally unique across all customers, and I don't see a quick way to customize this name via config. 

**Version:**
3

**Environment:**
OS X 10.12, node v7.1.0

**Steps to reproduce:**
 - Run `lambda setup` in a new environment. Probably on a different account than the developer that originally wrote this part of the code.

**Expected result:**
Expected no errors.

**Actual result:**
Error above.
</bug>

**Comment by henrinormak** [comment]
This is a known issue, can't promise an ETA on the fix unfortunately, as it requires some rethinking in terms of how infor such as the bucket name is stored

**Comment by zwily** [comment]
OK, but just to be crystal clear what this means: You are the only person in the world able to use lambda-tools until it's fixed.

**Comment by mklopets** [comment]
Not exactly. Lots of people under the same org/account can use lambda-tools. In production, even.  😜

**Comment by zwily** [comment]
Fair enough. Only people using your AWS account can use lambda-tools. :)

**Comment by henrinormak** [comment]
I have an idea, will try to fix this over the weekend.

**Comment by henrinormak** [comment]
Should be addressed in said PR, @mklopets will review, but @zwily you are more than welcome to also try it out before (clone and `npm link` in the directory should allow for lambda-tools to use local version instead of installed one).

**Comment by mklopets** [comment]
Issue was fixed in #61 and is released under version 3.1.0. You will need to re-run `lambda setup` with the `-p` flag to fix the issue on your machine. See the README for instructions.

Cheers!
